<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectDialog</class>
 <widget class="QDialog" name="CreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>242</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create project</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="LabelsLayout">
       <item>
        <widget class="QLabel" name="TitleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AuthorsLabel">
         <property name="text">
          <string>Authors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ProjectTypeLabel">
         <property name="text">
          <string>Project type</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="InputsLayout">
       <item>
        <widget class="QLineEdit" name="TitleLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="AuthoursLineEdit"/>
       </item>
       <item>
        <widget class="QComboBox" name="ProjectTypeComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="LocationLayout">
     <item>
      <widget class="QLineEdit" name="LocationLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="LocationToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ResultButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionChooseFolder">
   <property name="text">
    <string>Choose folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
